### PR TITLE
Linting

### PR DESCRIPTION
**Pull request by jhesketh**
Just some further tidy ups.

Use isort to be consistent with our imports.
Also run black against typing.

### CODE REVIEW

File: backend/typings/ceph_module.pyi [hunk 1]
```diff
@@ -3,113 +3,140 @@
...
-    def _pool_raw_used_rate(self, pool_id):...
+    def _dump(self): ...
+    def _new_incremental(self): ...
+    def _apply_incremental(self, inc: "BasePyOSDMapIncremental"): ...
+    def _get_crush(self): ...
+    def _get_pools_by_take(self, take): ...
+    def _calc_pg_upmaps(self, inc, max_deviation, max_iterations, pool): ...
+    def _map_pool_pgs_up(self, poolid): ...
+    def _pg_to_up_acting_osds(self, pool_id, ps): ...
+    def _pool_raw_used_rate(self, pool_id): ...
```
Comment (mgfritch) [inline backend/typings/ceph_module.pyi:24]: I think this file was generated using `stubgen` ...  maybe we should ignore the type errors in this file?

**Review by jecluis — APPROVED** [review]

File: backend/api/__init__.py [hunk 2]
```diff
@@ -11,8 +11,8 @@
 from fastapi import HTTPException, Request
 from fastapi.security import OAuth2PasswordBearer
 
-from bubbles.bubbles import Bubbles
 from bubbles.backend.auth import JWT, JWTDenyList, JWTMgr
+from bubbles.bubbles import Bubbles
```
[the remark this answers is not in the document]
Comment (jhesketh) [reply backend/api/__init__.py:15]: What do you expect it to do here? (sorry I'm confused)

File: backend/typings/ceph_module.pyi (hunk 1 above)
Comment (jhesketh) [reply backend/typings/ceph_module.pyi:24]: Ah, fair enough, I agree we should keep ignoring these then.

File: backend/api/__init__.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (jecluis) [reply backend/api/__init__.py:15]: nevermind, I misread that as `bubbles.bubbles.auth` instead of `bubbles.backend.auth`. Carry on, nothing to see here.

**Review by jecluis — APPROVED** [review]

**Pull request merged**